zenodo/base/format_templates/DataCite.xsl

```diff
@@ -124,6 +148,210 @@ exclude-result-prefixes="marc fn dc invenio">
                         <nameIdentifier nameIdentifierScheme="info">info:eu-repo/grantAgreement/EC/FP7/<xsl:value-of select="subfield[@code='c']"/></nameIdentifier>
                     </contributor>
                 </xsl:for-each>
+                <xsl:for-each select="datafield[@tag=700]">
+                    <xsl:if test="string-length(subfield[@code='4']) = 3 and subfield[@code='a']">
+                        <xsl:choose>
+                            <xsl:when test="subfield[@code='4']='prc'">
```

Comment:
Not sure what is the best way of providing these different contributors names.
I tried to put condition only on opening tag (which would reduce copying all body), but it didn't work for me, so for now I left it like that.